core/src/main/java/org/testcontainers/containers/DockerComposeContainer.java

```diff
@@ -194,4 +245,57 @@ public String getServiceHost(String serviceName, Integer servicePort) {
     public Integer getServicePort(String serviceName, Integer servicePort) {
         return ambassadorContainers.get(serviceName + ":" + servicePort).getMappedPort(servicePort);
     }
+
+    public SELF withScaledService(String serviceBaseName, int numInstances) {
+        scalingPreferences.put(serviceBaseName, numInstances);
+
+        return self();
+    }
+
+    public SELF withEnv(String key, String value) {
+        env.put(key, value);
+        return self();
+    }
+
+    public SELF withEnv(Map<String, String> env) {
+        env.forEach(env::put);
+        return self();
+    }
+
+    private SELF self() {
+        return (SELF) this;
+    }
 }
+
+class DockerCompose extends GenericContainer<DockerCompose> {
+    public DockerCompose(File composeFile, String identifier) {
+
+        super("dduportal/docker-compose:1.6.0");
```

Comment:
@valdisrigdon Good point, thanks. I've updated and other than a very minor tweak it seems to work fine.